cms/static/js/views/modals/move_xblock_modal.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * The MoveXblockModal to move XBlocks in course.
+ */
+define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview', 'js/views/modals/base_modal'
+], function(
+    $, Backbone, _, gettext, BaseView, BaseModal
+) {
+    'use strict';
+
+    var MoveXblockModal = BaseModal.extend({
+        events: _.extend({}, BaseModal.prototype.events, {
+            'click .action-move': 'moveXBlock'
+        }),
+
+        options: $.extend({}, BaseModal.prototype.options, {
+            modalName: 'move-xblock',
+            modalSize: 'med',
+            viewSpecificClasses: 'confirm',
+            title: gettext('Move'),
+            primaryActionButtonType: 'move',
+            primaryActionButtonTitle: gettext('Move')
+        }),
+
+        initialize: function() {
+            BaseModal.prototype.initialize.call(this);
+            this.template = this.loadTemplate('move-xblock-modal');
```

Comment:
I recommend that all new template access be done via RequireJS Text. I realize that the existing Studio code base doesn't do it this way, but it is much better in many dimensions (performance, modularity etc).